contracts/Lock.sol

```diff
@@ -8,10 +8,16 @@ contract Lock {
     uint256 public unlockTime;
     address payable public owner;
 
+    error InvalidUnlockTime(uint256 unlockTime);
+    error UnlockTimeNotReach(uint256 unlockTime);
+    error NotOwner(address owner);
```

Comment:
Can we order these alphabetically, please? And also define them outside of the contract?